docs/activities.md

```diff
@@ -13,17 +13,17 @@ import TabItem from '@theme/TabItem';
 
 This guide provides a comprehensive overview of Temporal Activities.
 
-In day-to-day conversations, the term _Activity_ frequently denotes either an <a class="tdlp" href="#activity-definition">Activity Definition<span class="tdlpiw"><img src="/img/link-preview-icon.svg" alt="Link preview icon" /></span><span class="tdlpc"><span class="tdlppt">What is an Activity Definition?</span><br /><br /><span class="tdlppd">An Activity Definition is the code that defines the constraints of an Activity Task Execution.</span><span class="tdlplm"><br /><br /><a class="tdlplma" href="#activity-definition">Learn more</a></span></span></a>, an <a class="tdlp" href="#activity-type">Activity Type<span class="tdlpiw"><img src="/img/link-preview-icon.svg" alt="Link preview icon" /></span><span class="tdlpc"><span class="tdlppt">What is an Activity Type?</span><br /><br /><span class="tdlppd">An Activity Type is the mapping of a name to an Activity Definition.</span><span class="tdlplm"><br /><br /><a class="tdlplma" href="#activity-type">Learn more</a></span></span></a>, or an <a class="tdlp" href="#activity-execution">Activity Execution<span class="tdlpiw"><img src="/img/link-preview-icon.svg" alt="Link preview icon" /></span><span class="tdlpc"><span class="tdlppt">What is an Activity Execution?</span><br /><br /><span class="tdlppd">An Activity Execution is the full chain of Activity Task Executions.</span><span class="tdlplm"><br /><br /><a class="tdlplma" href="#activity-execution">Learn more</a></span></span></a>.
+In day-to-day conversation, the term _Activity_ denotes an <a class="tdlp" href="#activity-definition">Activity Definition<span class="tdlpiw"><img src="/img/link-preview-icon.svg" alt="Link preview icon" /></span><span class="tdlpc"><span class="tdlppt">What is an Activity Definition?</span><br /><br /><span class="tdlppd">An Activity Definition is the code that defines the constraints of an Activity Task Execution.</span><span class="tdlplm"><br /><br /><a class="tdlplma" href="#activity-definition">Learn more</a></span></span></a>, <a class="tdlp" href="#activity-type">Activity Type<span class="tdlpiw"><img src="/img/link-preview-icon.svg" alt="Link preview icon" /></span><span class="tdlpc"><span class="tdlppt">What is an Activity Type?</span><br /><br /><span class="tdlppd">An Activity Type is the mapping of a name to an Activity Definition.</span><span class="tdlplm"><br /><br /><a class="tdlplma" href="#activity-type">Learn more</a></span></span></a>, or <a class="tdlp" href="#activity-execution">Activity Execution<span class="tdlpiw"><img src="/img/link-preview-icon.svg" alt="Link preview icon" /></span><span class="tdlpc"><span class="tdlppt">What is an Activity Execution?</span><br /><br /><span class="tdlppd">An Activity Execution is the full chain of Activity Task Executions.</span><span class="tdlplm"><br /><br /><a class="tdlplma" href="#activity-execution">Learn more</a></span></span></a>.
```

Comment:
Can you (or someone) mark all the generated files with this mechanism so the diffs show up as closed by default?
https://docs.github.com/en/repositories/working-with-files/managing-files/customizing-how-changed-files-appear-on-github

Reply:
awesome, will do, thanks!

Reply:
done. @flossypurse I added everything in docs/* that has a generated comment at the top. let me know if there are any more generated files

(another alternative is we could remove them from git)